src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/XUnitVerifier.cs

```diff
@@ -0,0 +1,154 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Can you put this file into Common somewhere?  It's not specific to regex.

Reply:
Might also be useful to include a link to the original source as a code comment.

Reply:
Yes, this is from the Roslyn SDK, and I just copied the source over. The Roslyn SDK contains a reference to xunit.assert, which was causing problems. However, my changes in https://github.com/dotnet/arcade/pull/13510 may make this obsolete.

The problems come from xunit.assert and Microsoft.DotNet.XUnitAssert having different assembly names -- so when both are included, all references are ambiguous. The alternative approach I'm proposing in that PR is to share the assembly name. Aside from making the XUnit analyzers work (which I think is a big benefit), it will remove these kinds of conflicts because I will no longer have to remove all shared references, and can instead rely on the Dotnet XunitAssert winning because it is higher version.

I'm ambivalent about the assembly name change, but after trying to fix a ton of these one-off issues, I'm warming up to it.

Reply:
IMO the assembly name changes is fine - we did the same with the Microsoft.DotNet.XUnitConsoleRunner: https://github.com/dotnet/arcade/blob/2d8d59065b5e090584a8e90c4371fc06ed60bdc5/src/Microsoft.DotNet.XUnitConsoleRunner/src/Microsoft.DotNet.XUnitConsoleRunner.csproj#L6

Avoiding renaming the assembly name between the upstream and the fork should be less breaking.